docs/README.md

```diff
@@ -12,3 +12,177 @@ We currently have the following components available for React applications:
+/>
+```
+
+### 3. Overriding styles with class names
+
+If you want to override a component's styles using custom classes, you can use the className prop, available on each component. To override the styles of a specific part of the component, use the global CSS classes provided, as described in the previous section "Overriding nested component styles".
+
+### 4. Styled component
+
+To reuse the same overrides in different locations across your application, create a reusable component using the [styled()](https://mui.com/system/styled/) utility.
+
+#### Usage
+
+```tsx
+import * as React from 'react';
+import { styled } from '@mui/material/styles';
+import { InfoListItem, InfoListItemProps } from '@brightlayer-ui/react-components';
+import { CheckCircle } from '@mui/icons-material';
+...
+const SuccessInfoListItem = styled(InfoListItem)<InfoListItemProps>(({ theme }) => ({
+    color: theme.palette.success.main,
+    '& .BluiInfoListItem-icon': {
+        color: theme.palette.success.main,
+    },
+}));
+
+export default function StyledCustomization() {
+  return  <SuccessInfoListItem
+              title={'Success'}
+              icon={<CheckCircle />}
+              statusColor={Colors.green[500]}
+          />;
+}
+```
+
+### 5. Overriding Styles with the `classes` prop
+
+To customize a specific part of a component, you can use the `classes` prop provided by Brightlayer UI. These classes can be found in the `Style Overrides` section of the documentation for each component. There are 4 ways you can approach using these:
+
+1. Continue using JSS until it is fully removed from @mui/material
+
+```tsx
+import createStyles from '@mui/styles/createStyles';
+import makeStyles from '@mui/styles/makeStyles';
+import { InfoListItem } from '@brightlayer-ui/react-components';
+import { Leaf } from '@brightlayer-ui/icons-mui';
+...
+const useStyles = makeStyles((theme: Theme) =>
+    createStyles({
+        icon: {
+            marginLeft: '2px',
+            fontSize: '1rem'
+        }
+    })
+);
+
+const classes = useStyles();
+
+<InfoListItem
+    title={'Info List Item'}
+    icon={<Leaf />}
+    classes={{ icon: classes.icon }}
+/>
+```
+
+2. Global style sheets
+
+ComponentFile.tsx
+
+```tsx
+import createStyles from '@mui/styles/createStyles';
+import makeStyles from '@mui/styles/makeStyles';
+import { InfoListItem } from '@brightlayer-ui/react-components';
+import { Leaf } from '@brightlayer-ui/icons-mui';
+...
+const useStyles = makeStyles((theme: Theme) =>
+    createStyles({
+        icon: {
+            marginLeft: '2px',
+            fontSize: '1rem'
+        }
+    })
+);
+
+const classes = useStyles();
+
+<InfoListItem
+    title={'Info List Item'}
+    icon={<Leaf />}
+    classes={{ icon: 'info-list-item-icon' }}
+/>
+```
+
+index.css
+
+```css
+.info-list-item-icon {
+    margin-left: 2px;
+    font-size: 1rem;
+}
+```
+
+3. [tss-react](https://mui.com/material-ui/migration/migrating-from-jss/#2-use-tss-react)
```

Comment:
Had to read three times before I saw this...it's really hidden in here with no description or example.